sfepy/examples/homogenization/nonlinear_hyperelastic_mM.py

```diff
@@ -1,7 +1,7 @@
 import numpy as nm
```

Comment:
It would be helpful to add 

`mpiexec -n 4 python3 simple.py --app=bvp-mM --debug-mpi sfepy/examples/homogenization/nonlinear_hyperelastic_mM.py`

hint to the docstring of this example?